.github/workflows/cross-version-tests.yml

```diff
@@ -114,7 +114,7 @@ jobs:
         run: |
           if [[ "${{ matrix.package }}" = "scikit-learn" && ! "${{ matrix.version }}" =~ ^0\.* ]] || \
              [[ "${{ matrix.package }}" = "statsmodels" ]] || \
-             [[ "${{ matrix.package }}" = "tensorflow" && "${{ matrix.version }}" = "dev" ]]
+             [[ "${{ matrix.package }}" = "tensorflow" && ! "${{ matrix.version }}" =~ ^2\.[0-6] ]]
```

Comment:
Would it be useful if we could specify python version to use in `ml-package-versions.yml`?

```yml
# ml-package-versions.yml

tensorflow:
  package_info:
    pip_release: "tensorflow"
    install_dev: |
      ...
    python_requires:
      ">= 2.7.0": "3.7"
```

Reply:
Yeah, it would be useful, but I don't think it's urgent